src/core/log.go

```diff
@@ -46,8 +46,14 @@ func getLogDirectory() string {
 	return filepath.Join(home, "YourPlace")
 }
 
-func LogInit(name string) *os.File {
-	logDir := getLogDirectory()
+func LogInit(name string, snapshotService bool) *os.File {
```

Comment:
we can do this better than adding a feature specific flag, to a feature generic code